targets/csharp/make.js

```diff
@@ -56,7 +54,9 @@ function getBaseTypeSyntax(datatype) {
 
     if (datatype.className.toLowerCase().endsWith("request"))
         parents.push("PlayFabRequestCommon");
-    if (datatype.className.toLowerCase().endsWith("response") || datatype.className.toLowerCase().endsWith("result"))
+    if (datatype.className.toLowerCase() === "loginresult")
```

Comment:
I realize that it wasn't you that created this problem, but let's fix it while you're here.
PlayFabRequestCommon, PlayFabLoginResultCommon, and PlayFabResultCommon are all classes.
Multi-Inheritance of classes is invalid in C#.
You can only inherit one class.

I think the best fix for now, is to wrap the "parent-classes" in some comments that say "begin classes - only 1 possible" and "end classes - only 1 possible", and make sure the if-switch inside only contains a single: if, elseif, elseif. 
We want to minimize the chance of a logic mistake producing two class-parents.

Reply:
Sorry I made a change but I didn't clearly understand your expectation. What i changed is ok for you? Please update me for your this and below comments